### DISCUSSION TITLE
In AWS MWAA v2.7.2, the displayed DAG counts include those that have been removed.

### DISCUSSION THREAD

**Discussion by ghost** [apache/airflow#36215] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

In AWS MWAA v2.7.2, the displayed DAG counts include those that have been removed. While this problem has been addressed in version 2.7.3, our current infrastructure is limited to supporting only version 2.7.2. Is deleting the inactive DAGs an option? Is there a viable method to fix this issue? If not, we welcome alternative suggestions for resolving this matter, as the current discrepancy in DAG counts is posing challenges in the efficient management of the DAGs. 

### What you think should happen instead

The DAG counts must be displayed correctly.

### How to reproduce

Deploy the DAGs and remove the DAG files.

### Operating System

Linux

### Versions of Apache Airflow Providers

apache-airflow-providers-google==10.9.0
apache-airflow-providers-snowflake==5.0.1

### Deployment

Amazon (AWS) MWAA

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.
